File: tarefa008/Bairro.ts
import { Cidade } from "./Cidade";

export class Bairro {
  constructor(nome:string) {
    this._nome = nome;
  }

  //atributos
  private _nome:string;
  private _cidade:Cidade;

  //setters
  set cidade(novaCidade:Cidade) {
    this._cidade = novaCidade;
  }
}

File: tarefa008/java/Bairro.java
package tarefa008.java;

/**
 * Classe que representa a entidade Bairro |
 * Autor: João Máiro |
 * Data: 15/07/22
 */
public class Bairro {
  Bairro(String nome) {
    this.nome = nome;
  }

  private String nome;
  private Cidade cidade;

  /**
   * Método que modifica o valor do atributo cidade
   * @param cidade
   */
  public void setCidade(Cidade cidade) {
    this.cidade = cidade;
  }

  /**
   * Método que retorna o valor do atributo nome
   * @return
   */
  public String getNome() {
    return this.nome;
  }

  /**
   * Método que retorna o valor do atributo cidade
   * @return
   */
  public Cidade getCidade() {
    return this.cidade;
  }
}